application/cms/views.py

```diff
@@ -450,54 +451,53 @@ def send_to_review(topic_uri, subtopic_uri, measure_uri, version):
     else:
         england = wales = scotland = northern_ireland = False
 
-    form_to_validate = MeasurePageRequiredForm(
+    measure_page_form_to_validate = MeasurePageRequiredForm(
```

Comment:
Would be nice to add the `sending_to_review=True` mechanism to this form too.  But not right now.

Reply:
Yep, agreed, I'd like to do that too. Will do it as a clean up thing later.